chart.tf

```diff
@@ -47,7 +48,7 @@ module "release" {
     },
     {
       name  = "config.MONGODB_HOST"
-      value = "${var.mongodb_host}"
+      value = jsondecode(data.aws_secretsmanager_secret_version.uri-host-secrets.secret_string)["host"]
```

Comment:
please have checks to allow both var.mongodb_host and jsondecode(data.aws_secretsmanager_secret_version.uri-host-secrets.secret_string)["host"] values be used depending on user passed variables?
apply this for all variables you changed bellow